web/test/pages/unsupported.test.tsx

```diff
@@ -0,0 +1,11 @@
+import UnsupportedPage from "@/pages/unsupported";
+import { render, screen } from "@testing-library/react";
+import Config from "../../../content/src/fieldConfig/config.json";
```

Comment:
for a test file, we never want to import Config directly anymore, with our new config pattern. we should do `const Config = getMergedConfig()` at the top of the test file.